pkg/component/runtime/manager_test.go

```diff
@@ -3481,6 +3481,164 @@ LOOP:
 	require.NoError(t, err)
 }
 
+func TestManager_FakeInput_Chunk(t *testing.T) {
+	testPaths(t)
+
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	grpcConfig := configuration.DefaultGRPCConfig()
+	grpcConfig.MaxMsgSize = 120 // very small size to test chunking
```

Comment:
This affects (and therefore tests) the expected direction correct?

Is there a way to force chunking on the checkin observed path as well?

Reply:
This affects both directions as the `MaxMsgSize` is sent to the client over stdin, so it will also use that max message size for the observed path.

Reply:
Ah I missed that part of this.